styles/globals.scss

```diff
@@ -31,7 +31,7 @@ body {
 
 h1 {
   font-weight: 600;
-  font-size: 42px;
```

Comment:
font should be dynamic for mobile viewing

Reply:
we have css that changes fontSize depending on how wide the screen is already